azure_monitor/azfunc_sample/README.md

```diff
@@ -1,90 +1,78 @@
+# Sample: Configuring Function Apps with Python and OpenCensus
 
-## Overview
+This is an example of how to add the various settings Python-based Azure Function applications to send dependencies calls to Azure Monitor.  You will run the samples both locally and deploy the Azure Functions to an Azure App Service resource you deployed as part of the setup script.
 
-Azure Function (Python) already supports instrumentation using Opencensus. The following sample utilizes the OpenCensus Functions extension and enables additional common telemetry scenarios used by Azure Monitor and Functions users. Such scenarios include:
-- End-to-end view of a single function invocation (in Azure Monitor Application Insights)
-- Create an application topology view (application dependencies to support distributed tracing) using Azure Monitor Application Insight's Application Map (especially useful when working with distributed, scalable applications). 
+## 1 - Environment Setup
 
-### Example of a Correlated End-to-End Azure Function Invocation
+Follow the steps in the [setup documentation](/azure_monitor/readme.md).
 
-End-to-end transaction:
+## 2 - Utilize Function Apps with OpenCensus
 
-![AI Correlated end2end function invocation](./docs/assets/01-ai-correlated-invocation.PNG)
+- Start the Azure Storage Emulator.
+- Switch to Visual Studio Code.
+- Open a terminal window, run the following commands to start the Python functions locally:
 
-Associated application trace messages:
+```powershell
+cd .\azure_monitor\azfunc_sample
 
-![AI Application trace messages](./docs/assets/02-ai-correlated-invocation.PNG)
+python -m pip install -r requirements.txt
 
-Associated AI Application Map:
+func start
+```
 
-![AI Application Map](./docs/assets/03-ai-application-map.PNG)
+- If you get an error about `psutil._psutil_windows` do the following:
+  - Open the `.venv\Lib\site-packages` folder and delete the `psutil` and `psutil-5.9.1.dist-info` folders.
+  - Then run the following:
 
-The images shows the following elements:
-
-- Correlated (per Azure Function invocation) log information including:
-    - Incoming request information
-    - Application traces & error messages
-    - External dependencies (IN-PROC & HTTP using Python's `requests` library)
-- Application Map (defined cloud role and cloud instance values)
-
-### Additional Customizations Enabling Azure Monitor Scenarios
+    ```Python
+    python -m pip install --upgrade pip
+    python -m pip install psutil
+    ```
 
-- In order to get the application topology (Application Insight's Application Map) the correct *Cloud Role Name* (field: cloud_RoleName) must be set (using a [Telemetry Processor](https://docs.microsoft.com/en-us/azure/azure-monitor/app/api-filtering-sampling#opencensus-python-telemetry-processors))
+- Browse to the HTTP function endpoint `http://localhost:7071/api/MyHttpFunction`.
 
-### Known Limitations in Current OpenCensus Azure Monitor Log Exporter
+## 3 - Deploy to Azure
```

Comment:
Why is deploying to azure a step within our docs for monitoring? Is this needed in your opinion @mattmccleary ?